jujugui/static/gui/src/app/store/env/go.js

```diff
@@ -2029,16 +2034,15 @@ YUI.add('juju-env-go', function(Y) {
       The parameters match the parameters for the public env deploy method in
       go.js.
 
-      @method setConfig
+      @method set_config
     */
-    set_config: function(serviceName, config, data, serviceConfig, callback,
-        options) {
+    set_config: function(service, config, callback, options) {
       var ecs = this.get('ecs');
       var args = ecs._getArgs(arguments);
       if (options && options.immediate) {
-        // Need to check that the serviceName is a real service name and not
+        // Need to check that the service is a real service name and not
         // a queued service id before allowing immediate or not.
-        if (ecs.changeSet[serviceName]) {
+        if (ecs.changeSet[service]) {
```

Comment:
I always think that `service` is the object. I'd prefer if this stayed its original name `serviceName` as it's more explicit about what it contains.